packages/contentstack-audit/src/types/content-types.ts

```diff
@@ -119,6 +120,7 @@ enum OutputColumn {
   title = 'title',
   'uid' = 'uid',
   'missingCts' = 'content_types',
+  'Missing Branches' = 'branches',
```

Comment:
Please fix this as well.!

Reply:
This one cannot be done as it's commonly used while creating the CSV file